source/features/select-notifications.tsx

```diff
@@ -150,10 +152,17 @@ function createDropdown(): JSX.Element {
 	);
 }
 
+function closeDropdown(): void {
+	select('.rgh-select-notifications[open] summary')?.dispatchEvent(new MouseEvent('click'));
```

Comment:
More compact, no events, simple selector

```suggestion
	select('.rgh-select-notifications')?.removeAttribute('open');
```